src/plugins/dashboard/server/dashboard_saved_object/dashboard_saved_object.ts

```diff
@@ -38,6 +41,33 @@ export const createDashboardSavedObjectType = ({
       };
     },
   },
+  modelVersions: {
+    1: {
+      changes: [],
+      schemas: {
+        forwardCompatibility: dashboardAttributesSchemaV1,
+        create: dashboardAttributesSchemaV1,
+      },
+    },
+    2: {
+      changes: [
+        {
+          type: 'mappings_addition',
+          addedMappings: {
+            controlGroupInput: {
+              properties: {
+                showApplySelections: { type: 'boolean', index: false },
+              },
+            },
+          },
+        },
+      ],
+      schemas: {
+        forwardCompatibility: dashboardAttributesSchemaV2,
```

Comment:
The forward compatibility schema should allow unknown fields (usually using `.extends({}, { unknowns: 'ignore' })`) since we don't know future structures and should be able to get back to old structures, e.g.:

https://github.com/elastic/kibana/blob/ca429ab8e7f1a4448c5e6de3c72cbedc30d90e69/src/plugins/saved_search/server/saved_objects/search.ts#L46

Same for `1`'s schema above :)

Reply:
Good call! Done in 6799dc96941484315bbf878d972f4bb030fb115b 🎉